services/intersection-api/jpo-conflictvisualizer-api/pom.xml

```diff
@@ -0,0 +1,368 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.1.3</version>
+		<relativePath /> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>usdot.jpo.ode</groupId>
+	<artifactId>jpo-conflictvisualizer-api</artifactId>
+	<version>1.4.0-SNAPSHOT</version>
+	<packaging>jar</packaging>
+	<name>jpo-conflictvisualizer-api</name>
+	<description>Conflict Visualizer</description>
+	<properties>
+		<java.version>21</java.version>
+        <jacoco.version>0.8.7</jacoco.version>
+        <sonar.java.coveragePlugin>jacoco</sonar.java.coveragePlugin>
+        <sonar.coverage.jacoco.xmlReportPaths>${project.basedir}/target/site/jacoco/jacoco.xml</sonar.coverage.jacoco.xmlReportPaths>
+        <sonar.language>java</sonar.language>
+		<!-- <thymeleaf.version>3.0.3.RELEASE</thymeleaf.version>
+    	<thymeleaf-layout-dialect.version>2.2.1</thymeleaf-layout-dialect.version> -->
```

Comment:
chore: either delete this comment or add some context as to when/why we would want to uncomment this dependency

Reply:
removed